Removing miniconda version 22.11.1-1
Loading miniconda version 22.11.1-1
loading libiconv version 1.16
loading xz version 5.2.5
loading zlib version 1.2.11
loading libxml2 version 2.9.12
loading cuda version 11.8.0
Traceback (most recent call last):
  File "/home/ctpham_umass_edu/.conda/envs/handbook/bin/accelerate", line 8, in <module>
    sys.exit(main())
  File "/home/ctpham_umass_edu/.conda/envs/handbook/lib/python3.10/site-packages/accelerate/commands/accelerate_cli.py", line 48, in main
    args.func(args)
  File "/home/ctpham_umass_edu/.conda/envs/handbook/lib/python3.10/site-packages/accelerate/commands/launch.py", line 1152, in launch_command
    args, defaults, mp_from_config_flag = _validate_launch_command(args)
  File "/home/ctpham_umass_edu/.conda/envs/handbook/lib/python3.10/site-packages/accelerate/commands/launch.py", line 986, in _validate_launch_command
    defaults = load_config_from_file(args.config_file)
  File "/home/ctpham_umass_edu/.conda/envs/handbook/lib/python3.10/site-packages/accelerate/commands/config/config_args.py", line 46, in load_config_from_file
    raise FileNotFoundError(
FileNotFoundError: The passed configuration file `recipes/accelerate_configs/deepspeed_zero3.yaml` does not exist. Please pass an existing file to `accelerate launch`, or use the default one created through `accelerate config` and run `accelerate launch` without the `--config_file` argument.
